configure-ironic.sh

```diff
@@ -62,6 +66,12 @@ else
     export IRONIC_INSPECTOR_BASE_URL="http://${IRONIC_URL_HOST}:5050"
 fi
 
+if [ -f "$MARIADB_CERT_FILE" ] && [ -f "$MARIADB_CACERT_FILE" ]; then
```

Comment:
If you are not default the CA to the actual cert, then you should remove the cert file from this condition, as the ca file would be empty

Reply:
We don't need to do that. If Ironic cannot find the MariaDB CA certificate using the provided link, it will automatically connect without TLS. This behaviour has been tested

Reply:
I was not clear with my comment, sorry. You are not using the cert file in ironic, because you use the CA. Why would you require to have both the cert itself and the CA when you use only the CA. (I meant cert file, not CA file in the previous comment). You are putting a dependency on a file that you do not use. If you remove this then you don't need to mount the cert secret at all, just the CA

Reply:
Oh I get your idea. I thought you were saying that we should delete the certificate file. Sorry for that. Updated.